### PR TITLE
Update MetaMask connector doc

### PR DESCRIPTION
**Pull request by alexandratran**
This PR updates the doc page for the MetaMask SDK connector. This includes removing several options that we don't recommend configuring.

### CODE REVIEW

**Comment by tmm** [comment]
@alexandratran is this ready for merge? looks good from my perspective.

**Review by EdouardBougon — CHANGES_REQUESTED** [review]

File: site/shared/connectors/metaMask.md [hunk 1]
```diff
@@ -36,133 +36,46 @@ export const config = createConfig({
...
-
-`WakeLockStatus | undefined`
+```ts-vue
+import { metaMask } from '{{connectorsPackageName}}'
 
-The type of wake lock to use when the SDK is running in the background.
+const connector = metaMask({
+  headless: true // [!code focus]
+})
+```
```
Comment (EdouardBougon) [inline site/shared/connectors/metaMask.md:81]: ## Advanced
By default, this connector will replace the EIP_6963 MetaMask injected connector if it is detected (See the option [`rdns`](https://wagmi.sh/dev/creating-connectors#properties))

**Review by EdouardBougon — CHANGES_REQUESTED** [review]

File: site/shared/connectors/metaMask.md [hunk 2]
```diff
@@ -43,8 +43,13 @@ Check out the [MetaMask SDK docs](https://docs.metamask.io/wallet/connect/3rd-pa
 
 `DappMetadata | undefined`
 
-Metadata about the dapp using the SDK, including `name`, `url`, and `iconUrl`.
-By default, `name` is set to `'wagmi'`.
+Metadata about the dapp using the SDK, including:
+
+- `name` - The name of the dapp.
+  The default is `'wagmi'`.
```
Comment (EdouardBougon) [inline site/shared/connectors/metaMask.md:49]: I prefer to remove this line. The default "wagmi" value is only on a specific case, when the value `window.location.origin` is not defined. I don't want to document this edge case.

File: site/shared/connectors/metaMask.md [hunk 3]
```diff
@@ -43,8 +43,13 @@ Check out the [MetaMask SDK docs](https://docs.metamask.io/wallet/connect/3rd-pa
...
+- `url` - The URL of the dapp.
+  The default is `'window.location.origin'`.
```
Comment (EdouardBougon) [inline site/shared/connectors/metaMask.md:51]: The default is `window.location.origin` value.

**Comment by alexandratran** [comment]
@tmm This should be ready to merge now if it looks good to you!

**Pull request merged**